docs/INSTALL.md

```diff
@@ -179,19 +179,26 @@ lnd --bitcoin.active --bitcoin.testnet --debuglevel=debug --btcd.rpcuser=kek --b
 
 #### Running lnd using the bitcoind backend
 
-In order to run `lnd` with a `bitcoind` back-end, the `bitcoind` instance must
-be configured with `--txindex` just like `btcd` above. In addition, you'll need
- to configure the `bitcoind` instance with `--zmqpubrawblock` and `--zmqpubrawtx`
+To configure your bitcoind backend for use with lnd, first complete and verify the following:
+
+- The `bitcoind` instance must
+be configured with `--txindex` just like `btcd` above
+- Additionally, since `lnd` uses [ZeroMQ](https://github.com/bitcoin/bitcoin/blob/master/doc/zmq.md) to interface with `bitcoind`, *your `bitcoind` installation must be compiled with ZMQ*. If you installed it from source, this is likely the case, but if you installed it via Homebrew it may not be included (a [PR is in progress](https://github.com/Homebrew/homebrew-core/pull/23088) for this issue)
+- Configure the `bitcoind` instance for ZMQ with `--zmqpubrawblock` and `--zmqpubrawtx`
 (the latter is optional but allows you to see unconfirmed transactions in your
-wallet). They must be combined in the same ZMQ socket address. Then, run this
-command after `bitcoind` has finished syncing on testnet. Otherwise, replace
-`--bitcoin.testnet` with `--bitcoin.regtest`. Please note that the `rpcuser`
-and `rpcpass` parameters can typically be determined by `lnd` for a `bitcoind`
-instance running under the same user, including when using cookie auth.
+wallet). They must be combined in the same ZMQ socket address (e.g. `--zmqpubrawblock=tcp://127.0.0.1:28332` and `--zmqpubrawtx=tcp://127.0.0.1:28332`).
+- Start `bitcoind` running against testnet, and let it complete a full sync with the testnet chain (alternatively, use `--bitcoind.regtest` instead).
+
+Once all of the above is complete, and you've confirmed `bitcoind` is fully updated with the latest blocks on testnet, run the command below to launch `lnd` with `bitcoind` as your backend (as with `bitcoind`, you can create an `lnd.conf` to save these options, more info on that is described further below):
+
 ```
-lnd --bitcoin.active --bitcoin.testnet --debuglevel=debug --bitcoin.node=bitcoind --bitcoind.rpcuser=kek --bitcoind.rpcpass=kek --externalip=X.X.X.X
+lnd --bitcoin.active --bitcoin.testnet --debuglevel=debug --bitcoin.node=bitcoind --bitcoind.rpcuser=REPLACEME --bitcoind.rpcpass=REPLACEME --bitcoind.zmqpath=tcp://127.0.0.1:28332 --externalip=X.X.X.X
```

Comment:
I was just about to make a docs PR for using `bitcoind.rpcuser` and `bitcoind.rpcpass` as well.  Glad it's already done!

Reply:
:)